.changeset/shiny-paws-chew.md

```diff
@@ -0,0 +1,5 @@
+---
+"fuels": patch
+---
+
+feat: add minified `fuels` distribution
```

Comment:
Is this still a WIP?

Reply:
Just wanted to test the minified distribution